.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,51 @@
+name: Build and Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+          # - macos-latest
+          # - windows-latest
```

Comment:
Do we want to run the action on `macos` and `windows` too?

Reply:
Ideally we would do at least one build on each, like 4.11.1 for example